src/api/app/controllers/distributions_controller.rb

```diff
@@ -23,7 +23,7 @@ def index
 
   # GET /distributions/1234
   def show
-    @distribution = Distribution.find(params[:id]).to_hash
+    @distribution = Distribution.find(params[:id])
```

Comment:
If you move this into a `before_action` you can reuse it below and fix the code climate issues about `destroy` action too complex

Reply:
:monocle_face: the codeclimate issue is about `RepositoriesController`

Reply:
So true! Forget it, then.